Holovibes/sources/core/pipe.cc

```diff
@@ -398,7 +398,7 @@ void Pipe::insert_moments()
     if (recording || type == ImgType::Moments_0 || type == ImgType::Moments_1 || type == ImgType::Moments_2)
     {
         auto p = setting<settings::P>();
-        moments_env_.f_start = p.start;
+        moments_env_.f_start = p.start - 1; // -1 because indexes start at 0 in C++
```

Comment:
I assume the `-1` is to switch to base 0. You should also modify in `tools_conversion.cu` the function `insert_compute_p_accu` that compute pmin and pmax (based on Z).